### ISSUE TITLE
stored procedure out parameters

### ISSUE THREAD

**Issue by Idhor** [ziutek/mymysql#108]
Hi,
is there a way to read output parameters of stored procedures?
Thank you!

**Comment by ziutek** [comment]
Yes. See https://github.com/ziutek/mymysql#example-8---use-stored-procedures
